chapters/story.md

```diff
@@ -215,16 +217,257 @@ After determining the need for a concept set defining anemia, which is a common
+        <th>Death</th>
+        <th>Time_to_Death</th>
+    </tr>
+    <tr>
+        <td>27</td>
+        <td>M</td>
+        <td>Non-Hispanic White</td>
+        <td>Q2 2022</td>
+        <td>Current or Former Smoker</td>
+        <td>0</td>
+        <td>0</td>
+        <td>0</td>
+        <td>0</td>
+        <td>1</td>
+        <td>0</td>
+        <td>0</td>
+        <td>0</td>
+        <td>1</td>
+        <td>0</td>
+        <td>28</td>
+        <td>0</td>
+        <td>28</td>
+    </tr>
+    <tr>
+        <td>75</td>
+        <td>F</td>
+        <td>Hispanic or Latinx</td>
+        <td>Q1 2021</td>
+        <td>No Documented History of Smoking</td>
+        <td>0</td>
+        <td>0</td>
+        <td>0</td>
+        <td>0</td>
+        <td>0</td>
+        <td>0</td>
+        <td>1</td>
+        <td>0</td>
+        <td>0</td>
+        <td>0</td>
+        <td>28</td>
+        <td>0</td>
+        <td>28</td>
+    </tr>
+    <tr>
+        <td>54</td>
+        <td>M</td>
+        <td>Non-Hispanic Black or African American</td>
+        <td>Q4 2020</td>
+        <td>No Documented History of Smoking</td>
+        <td>1</td>
+        <td>1</td>
+        <td>0</td>
+        <td>1</td>
+        <td>0</td>
+        <td>0</td>
+        <td>1</td>
+        <td>0</td>
+        <td>1</td>
+        <td>0</td>
+        <td>1</td>
+        <td>1</td>
+        <td>1</td>
+    </tr>
+    <tr>
+        <td>34</td>
+        <td>F</td>
+        <td>Non-Hispanic White</td>
+        <td>Q1 2023</td>
+        <td>Current or Former Smoker</td>
+        <td>1</td>
+        <td>0</td>
+        <td>1</td>
+        <td>1</td>
+        <td>0</td>
+        <td>1</td>
+        <td>0</td>
+        <td>0</td>
+        <td>1</td>
+        <td>1</td>
+        <td>3</td>
+        <td>1</td>
+        <td>5</td>
+    </tr>
+    <tr>
+        <td>…</td>
+        <td>…</td>
+        <td>…</td>
+        <td>…</td>
+        <td>…</td>
+        <td>…</td>
+        <td>…</td>
+        <td>…</td>
+        <td>…</td>
+        <td>…</td>
+        <td>…</td>
+        <td>…</td>
+        <td>…</td>
+        <td>…</td>
+        <td>…</td>
+        <td>…</td>
+        <td>…</td>
+        <td>…</td>
+    </tr>
+</table>
+</div>
 
 ## Analyses
 
-* Developing the Analyses
-* finalizing analysis
-  * Pinning to a release
-  * DRR
-  * figures
+:::{.callout-note icon=false}
+
+## Voice of Narrator
 
+Once the analytic dataset is finalized, the handoff between the informaticist and statistician takes place. This often requires a discussion or ongoing dialogue to understand the data structure and definitions. 
+:::
+
+* **Navigator**: Welcome everyone. Today we'd like to review the analytic dataset and next steps in moving the analysis forward. 
+* **Informaticist**: Yep. I've made a new code workbook in your folder in the project workspace. Can you pull it up and we can take a look.
+* **Statistician**: Sure. Did you send me the dataset? I don't remember seeing an email. 
```

Comment:
@jerrodanzalone, I love this exchange, even though my people are the butt of the joke.  Very representative.